### PR TITLE
[WIP]Lithuanian language addition

### PR DESCRIPTION
**Pull request by Pionell**
Translated the Bolt cms to Lithuanian a while ago. This was translated on version 3.3.6, and I haven't had time to test it on newer releases.

### CODE REVIEW

File: app/resources/translations/lt_LT/validators.lt_LT.xlf [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" datatype="plaintext" original="file.ext">
+        <body>
+            <trans-unit id="1">
+                <source>Slaptažodžiai nesutampa.</source>
```
Comment (horus68) [inline app/resources/translations/lt_LT/validators.lt_LT.xlf:6]: "source" should not be translate, only "target".
Original source is "Passwords must match." but you translated the opposite?
Please see https://crowdin.com/project/bolt

**Review by horus68 — CHANGES_REQUESTED** [review]

File: app/resources/translations/lt_LT/infos.lt_LT.yml [hunk 2]
```diff
@@ -0,0 +1,127 @@
+info:
+    upload:
+        image: |
+            Šis laukelis naudojamas nuotraukos ar paveiksliuko įkėlimui. Paspauskite mygtuką failo įkėlimui iš kompiuterio arba, jei naudojate naujausią Chrome arba Firefox versiją, galite tiesiog nutempti failus iš darbalaukio arba kito naršyklės lango.<br>
+            Taip pat galite naudoti anksčiau įkeltą paveikslėlį. Kad pasirinktumėte anksčiau įkeltą failą, tiesiog parašykite failo pavadinimą (ar jo dalį) įvesties laukelyje ir failas bus automatiškai įkeltas.
+            
+            Use this field to upload a photo or image. Click the button to upload a file from your
```
Comment (horus68) [inline app/resources/translations/lt_LT/infos.lt_LT.yml:7]: Strange to see english values present here along with your language!

File: app/resources/translations/lt_LT/validators.lt_LT.xlf (hunk 1 above)
Comment (Pionell) [reply app/resources/translations/lt_LT/validators.lt_LT.xlf:6]: what do you mean by opposite? "slaptažodžiai nesutampa" more or less == "Passwords must match." just depends on the way you translator interprets it, but it's an accurate translation

File: app/resources/translations/lt_LT/infos.lt_LT.yml (hunk 2 above)
Comment (Pionell) [reply app/resources/translations/lt_LT/infos.lt_LT.yml:7]: whoops, missed this one

File: app/resources/translations/lt_LT/validators.lt_LT.xlf (hunk 1 above)
Comment (jadwigo) [reply app/resources/translations/lt_LT/validators.lt_LT.xlf:6]: @horus68 means it should be:
```
<source>Passwords must match.</source>
<target>Slaptažodžiai nesutampa.</target>
```
The `<source>` probably needs to be the original english text

Comment (Pionell) [reply app/resources/translations/lt_LT/validators.lt_LT.xlf:6]: oh, ok, sorry, will modify it :)

**Comment by bobdenotter** [comment]
@horus68 In slightly related news: I've made a start on getting the translations for Bolt 4 up on Crowdin. It's not much just yet, but it's a start :-) 

https://crowdin.com/project/bolt/settings#files

**Comment by horus68** [comment]
@bobdenotter good to know!
Note: you can use the branch feature in Crowdin to have both v3 and v4 files there!

**Pull request closed without merging**